roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -32,6 +32,17 @@
   with_items: "{{object_def_files.files | map(attribute='path') | list | sort}}"
   changed_when: no
 
+- oc_obj:
+    state: absent
+    name: "{{item.name}}"
+    kind: "{{item.kind}}"
+    kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
+    namespace: "{{openshift_logging_namespace}}"
+  with_items:
+  - {name: logging-kibana, kind: route}
+  - {name: logging-kibana-ops, kind: route}
+  - {name: kibana-proxy, kind: oauthclient}
```

Comment:
@detiber @ewolinetz This was what I was trying to highlight in the feature; the only way to replace a route where the host has changed is to delete it and recreate it.  The hostname is a fairly easy check and just a little more work; I'll proceed doing something here.  I'm not certain to what degree we want to check the other parts change.  It may not matter if using oc apply but then that will fail if the hostname and certs change.  Do we wish to get that involved?  Is changed that important to introduce the complexity?

The oauthclient produced an error because it needed the resource version for 'apply' to work.  My concern is if the version changes between generation and apply.  Is that a risk we need to accept and just add the resourceversion to the object def?